assets/travis-worker/clean-up-containers.bash

```diff
@@ -37,13 +37,8 @@ main() {
     local cn
     cn="$(docker inspect "${cid}" --format '{{ .Name }}')"
 
-    if [[ "${cn}" == /travis-worker ]]; then
-      not_killed_count="$((not_killed_count + 1))"
-      continue
-    fi
-
-    if ! __container_is_newer_than "${cid}" "${max_age}"; then
-      __logger info "${cid} is older than ${max_age}; removing! (name=${cn})"
+    if __container_must_be_cleaned "${cid}" "${max_age}"; then
+      __logger info "${cid} must be cleaned; removing! (name=${cn})"
 
       docker kill "${cid}" ||
```

Comment:
You're getting container IDs below via `docker ps --all` which will include any `Exited` containers that may be present. Running `docker kill` on such a container will result in an error like:
```
$ docker kill c0606705324d
Error response from daemon: Cannot kill container: c0606705324d: Container c0606705324de62755c03ae45e649cb745754607dea39f2e24239f8dbbe6309d is not running
```

Right?
I figure that might cause some confusion since it will result in the same error message (`failed to kill container name=${cn}`) as if it fails for some other reason. But maybe you're ok with that :)

Reply:
Yep, it's definitely sloppy, but I mostly don't care (?) :grimacing: